app/api/graduations/me/func.js

```diff
@@ -0,0 +1,10 @@
+const app = require('@/app')
+
+module.exports = async function (context) {
+  const historiesGraduationIds = await app.models.historiesGraduations.find({ ra: context.user.ra}).select("_id")
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555380)
```suggestion
  const historiesGraduationIds = await app.models.historiesGraduations.find({ ra: context.user.ra}).select('_id')
```